ppr-ui/src/views/mhrInformation/MhrInformation.vue

```diff
@@ -715,11 +743,15 @@ export default defineComponent({
     }
 
     const scrollToFirstError = async (scrollToTop: boolean = false): Promise<void> => {
+      const mappedSections = mapTransferReviewValidations()
+      // Set the first invalid section
+      const invalidSection = document.getElementById(mappedSections.filter(item => item.value === false)[0].section)
+
       setTimeout(() => {
         scrollToTop
           ? document.getElementById('mhr-information-header').scrollIntoView({ behavior: 'smooth' })
           : document.getElementsByClassName('border-error-left').length > 0 &&
-            document.getElementsByClassName('border-error-left')[0].scrollIntoView({ behavior: 'smooth' })
+            invalidSection.scrollIntoView({ block: 'start', inline: 'nearest', behavior: 'smooth' })
```

Comment:
I don't think you will need the short circuit protection here anymore, with the refactors.